sdk/metric/version.go

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric // import "go.opentelemetry.io/otel/sdk/metric"
+
+// version is the current release version of the metric SDK in use.
+func version() string {
```

Comment:
shouldn't this function be public?

Reply:
@yurishkuro this is intended as per [this comment](https://github.com/open-telemetry/opentelemetry-go/issues/2143#issuecomment-889468513)

Reply:
I don't understand the purpose of this function if it's private as you have it now. The ticket had a discussion about a public `Version()` function that could be called externally.

Reply:
@yurishkuro i agree the ticket was for a public Version() function but the comment by @Aneurysm9 

"We did not have any readily available use case for exporting the version information, so the function should be unexported until such time as requirements dictate otherwise." 

I am a little confused on how to proceed with this now..

Reply:
> I don't understand the purpose of this function if it's private as you have it now. The ticket had a discussion about a public `Version()` function that could be called externally.

Even if this function is not exported it can be useful to report the version of this module in resource information.  The build tooling will ensure that it is updated as we make releases and it provides a single point to make that change.  If we find that there is need for it to be exported then that is an easy change.